### DISCUSSION TITLE
Don't we see a laser here?

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#290] [General]
From [this post](https://github.com/orgs/a-cosmology-group/discussions/281#discussioncomment-11322126)

> Hmm .. [from another thread](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-11066623) and adapted a bit. Don't we see a laser here? - @HanDeBruijn

![laserjet](https://github.com/user-attachments/assets/1f44bd14-c84e-4630-861f-6d185eb54ed6)

@HanDeBruijn, the image shows a plasma confined to a "straight" magnetic line, not a laser.  The stronger confinement (left) makes it tighter, hotter, and brighter.
  The plasma emits light that traces the magnetic line.

Another observation: On the right-hand side, the magnetic line has a slight downward curve inside the plasma emitting the radio lobe.
  A light beam would not curve like that.

**Comment by LifeIsStrange** [comment]
off topic but weird imaging of the closest quasar from Hubble
https://phys.org/news/2024-12-hubble-closest-quasar.html

**Reply by HanDeBruijn** [reply]
Off topic? How so? Didn't we have a lot of quasar / jet / drain pictures in the [_white dwarfs &amp; black dwarfs_](https://github.com/orgs/a-cosmology-group/discussions/144) thread lately?

**Reply by budrap00** [reply]
A typical Phys.org piece of dogmatic drivel in which good quasar observational evidence is crammed into the SMBS of the imaginary SMBH fairy tale. The evidence would be a lot more interesting without the SMBH baggage which is just there to sustain the LCDM fairy tale. It is hard to overstate how LCDM orthodoxy obstructs a rational understanding of astrophysical observations.
